Wikipedia/mw-bridge/CommunicationBridge.m

```diff
@@ -26,6 +27,7 @@ - (CommunicationBridge*)initWithWebView:(UIWebView*)targetWebView;
 
         __weak CommunicationBridge* weakSelf = self;
         [self addListener:@"DOMContentLoaded" withBlock:^(NSString* type, NSDictionary* payload) {
+            [weakSelf.webView wmf_enableJavascriptToXcodeConsoleLogging];
```

Comment:
for example:

``` objc
JSValue* console = [[self wmf_javascriptContext] globalObject][@"console"];
@weakify(self);
console[@"log"] = ^(NSString* message) {
    @strongify(self);
    [self wmf_logWebViewMessage:message];
};
```

could use this same approach to add `window.onerror` handler like I mentioned

Allows for all console log messages to show up:
```
(lldb) expr (void) [[[0x7ba30200 bridge] webView] stringByEvaluatingJavaScriptFromString:@"console.log('hello world!');"]
2015-08-25 09:49:30:867 Wikipedia Debug[main] -[UIWebView(WMFJavascriptToXcodeConsoleLogging) wmf_logWebViewMessage:]#L20 INFO:
hello world!
```